.env.default

```
@@ -1,3 +1,4 @@
 # The endpoint to the root of the API
 NEXT_PUBLIC_API_ROOT=https://api.indiavaccine.in/api/
 NEXT_PUBLIC_GA_TRACKING_ID=
+NEXT_PUBLIC_GEOCODING_API_KEY=643c1e6973c7427092debb04555cc381
```

Best not to expose this key

```suggestion
NEXT_PUBLIC_GEOCODING_API_KEY=
```